crates/bevy_utils/macros/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+use proc_macro::TokenStream;
```

Comment:
```suggestion
#![forbid(unsafe_code)]
#![warn(missing_docs)]

use proc_macro::TokenStream;
```

Let's try to enforce this from the start.